sklearn/linear_model/passive_aggressive.py

```diff
@@ -110,6 +121,16 @@ def partial_fit(self, X, y, classes=None):
         -------
         self : returns an instance of self.
         """
+        if self.class_weight == 'balanced':
+            raise ValueError("class_weight 'balanced' is not supported for "
+                             "partial_fit. In order to use 'balanced' "
+                             "weights, from the sklearn.utils module use "
+                             "compute_class_weight('balanced', classes, y). "
+                             "In place of y you can us a large enough sample "
```

Comment:
you can us -> you can use
I'd say "subset" instead of "sample", because it's less ambiguous. (consider `n_samples`).